### PR TITLE
Chores/merge with example client 08 26

### PR DESCRIPTION
**Pull request by shannonwells**
Purpose
---------------
Merge with changes from example-client. @claireolmstead  please especially look at the profiles code and the CSS. I'm sure some other things aren't quite right. I made some best guesses on what UI stuff is done in Foodee, but the feed is showing up, and you can login and back out. The populate script runs - I didn't bring in any of the account data to the populate script, because I knew your PR has the account data we want. Profile images aren't showing though.

### CODE REVIEW

File: src/components/Login.tsx [hunk 1]
```diff
@@ -1,13 +1,16 @@
 import React from "react";
-import { Alert, Dropdown, Button } from "antd";
+import { Badge, Button, Dropdown } from "antd";
```
Comment (shannonwells) [inline src/components/Login.tsx:2]: In this file I took example-client code and merged the components to try to get Login looking like Foodee and working like example-client

File: src/components/UserAvatar.tsx [hunk 2]
```diff
@@ -25,6 +25,7 @@ const UserAvatar = ({
     : blockies.create({ seed: profileAddress }).toDataURL();
   return (
     <Avatar
+      className="UserAvatar"
```
Comment (shannonwells) [inline src/components/UserAvatar.tsx:28]: don't know if this is needed

File: src/components/scss/Login.scss [hunk 3]
```diff
@@ -23,3 +23,8 @@
   align-items: flex-start;
   padding-left: 15px;
 }
+
+.Login__walletIcon {
```
Comment (shannonwells) [inline src/components/scss/Login.scss:27]: also don't know if the walletIcon is needed - if not we can take this styling out.

**Comment by claireclark1** [comment]
I would love if this could get in before my current PR because it resolves some of the issues that Shannon is talking about.

File: src/components/scss/Login.scss (hunk 3 above)
Comment (claireclark1) [reply src/components/scss/Login.scss:27]: taken out in next PR

**Review by claireclark1 — APPROVED** [review]

**Pull request merged**